RocketData/CollectionDataProvider.swift

```diff
@@ -295,7 +295,7 @@ open class CollectionDataProvider<T: SimpleModel>: ConsistencyManagerListener, B
      - parameter shouldCache: If false, we will not persist this to the cache.
      - parameter context: This context will be passed onto the cache delegate. Default nil.
      */
-    open func removeAtIndex(_ index: Int, shouldCache: Bool = true, context: Any? = nil) {
+    open func remove(at index: Int, shouldCache: Bool = true, context: Any? = nil) {
```

Comment:
I don't think so based off Apple's APIs. Apple specifically suggests against this in API guidelines https://swift.org/documentation/api-design-guidelines/ (Omit needless words)

from ^
bad `public mutating func removeElement(_ member: Element) -> Element?`
good `public mutating func remove(_ member: Element) -> Element?`